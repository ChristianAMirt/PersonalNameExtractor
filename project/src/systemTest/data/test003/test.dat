<NER>The excited Emile Hesby waited for the letter written by Lucio Mowen. The letter contained a signature of approval for his future endeavors with Rebecca Skipworth.</NER> 