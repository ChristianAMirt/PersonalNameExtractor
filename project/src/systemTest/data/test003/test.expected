<NER>The excited <PER> Emile Hesby </PER> waited for the letter written by <PER> Lucio Mowen </PER> . The letter contained a signature of approval for his future endeavors with <PER> Rebecca Skipworth </PER> .</NER> 
